addons/web/static/src/core/popover/popover.js

```diff
@@ -61,10 +61,11 @@ export class Popover extends Component {
             validate: (target) => {
                 // target may be inside an iframe, so get the Element constructor
                 // to test against from its owner document's default view
-                const Element = target?.ownerDocument?.defaultView.Element;
+                const Element = target?.ownerDocument?.defaultView?.Element;
                 return (
-                    Boolean(Element) &&
-                    (target instanceof Element || target instanceof window.Element)
+                    (Boolean(Element) &&
+                        (target instanceof Element || target instanceof window.Element)) ||
+                    (typeof target === "object" && target?.constructor?.name?.endsWith("Element"))
```

Comment:
what's the usecase?

Reply:
just this being late: https://github.com/odoo/odoo/pull/151246